ansible/roles/probe-services/tasks/main.yml

```diff
@@ -31,6 +31,11 @@
     src: ooni-orchestrate.toml
     dest: "{{ orchestrate_data_dir }}/ooni-orchestrate.toml"
 
+- name: Write Psiphon config file
+  copy:
+    content: "{{ orchestra_psiphon_config_file_content }}"
```

Comment:
You need to also defined this variable inside of the `defaults/main.yml` and probably set it to the `{{ CHANGE_ME }}` so that if you by mistake forget to define it in the scope of the target host, it will trigger an error.

If you don't define a default for it, when the role is getting compiled (which is before it's **evaluated** with the target host which is inside of the `psk_orchestration_prod` group) you will get an error.

Reply:
50a52b9